build/config.gni

```diff
@@ -24,6 +24,10 @@ declare_args() {
   chrome_version_major = ""
   build_omaha = false
   tag_ap =""
+
+  # Always disable building Cloud Print and Chromium PDF Viewer
+  enable_printing = false
```

Comment:
this is actually creating two new and potentially conflicting values. I'm surprised gn doesn't complain about it. Overriding of existing gn config values is done in brave-browser https://github.com/brave/brave-browser/blob/master/lib/config.js#L65

Reply:
I see. Would it be ok if add a comment describing such? I think most people would start at `build/config.gni` the way it is done in chromium, and then got tripped.

Reply:
the `declare_args` sections are just that, a declaration of arguments. In Chromium the default values are defined there and overrides use args.gn. config.js automatically generates the args.gn file for Brave. So we are doing things exactly like Chromium, we just have a helper to generate the args.gn file

Reply:
see https://www.chromium.org/developers/gn-build-configuration

Reply:
so this is the difference between creating new args and overriding existing args